cpp/include/Ice/Initialize.h

```diff
@@ -661,6 +662,32 @@ class ICE_API CommunicatorHolder
     CommunicatorPtr _communicator;
 };
 
+/**
+ * The output mode for xxxToString method such as identityToString and proxyToString. The actual encoding format for
+ * the string is the same for all modes: you don't need to specify an encoding format or mode when reading such a
+ * string.
+ */
+enum class ToStringMode : unsigned char
```

Comment:
Moved here to "fix" a circular dependency between Communicator.h and Initialize.h.